apps/builder/app/builder/features/sidebar-left/panels/pages/page-settings.tsx

```diff
@@ -1524,8 +1524,9 @@ const updatePage = (pageId: Page["id"], values: Partial<Values>) => {
         throw new Error(`Page with id ${pageId} not found`);
       }
 
-      const oldHomePage = pages.homePage;
-      pages.homePage = pages.pages[newHomePageIndex];
+      const oldHomePage = pages.homePage as (typeof pages.pages)[0];
```

Comment:
Weird, type should've been the same. What is changed? Does refine narrows types now?

Reply:
Yes